lib/ralyxa/response_entities/card.rb

```diff
@@ -20,53 +20,53 @@ def self.link_account
       end
 
       def to_h
-        Hash.new.tap do |card|
-          set_type(card)
-          set_title(card) if @options[:title]
-          set_body(card)  if @options[:body]
-          set_image(card) if @options[:image_url]
+        {}.tap do |card|
+          add_type(card)
+          add_title(card) if @options[:title]
+          add_body(card)  if @options[:body]
+          add_image(card) if @options[:image_url]
```

Comment:
Thank you; much nicer naming

Reply:
More then welcome, the style guide has a smell for methods beginning with `set_` and `add` felt like a good alternative :+1: